app/javascript/mastodon/emojione_light.js

```diff
@@ -1,13 +1,40 @@
 // @preval
```

Comment:
I don't believe it's valuable to keep using `@preval`. I tested with and without it, and because the emoji data is duplicated, `common.js` is 125kB smaller if we remove `@preval` (1463880 bytes vs 1338993 bytes).

Also I tested on Chrome on a Nexus 5, and the time to run this code client-side is only 35 milliseconds. So it's not a good tradeoff IMO.